docs/template-vm.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+    miq.github.io/kubevirt-is-vm-template: true
```

Comment:
That should be a label, so that manageiq can directly query for supported templates.

Reply:
ok, moving as label